### PR TITLE
feat(menu-toggle): added plain w/text variant

### LINKED ISSUE #4407: Menu Toggle: Support for plain variant with text
The Options menu supports a plain variation with text.  We need this in order to build this out with the compassable menu.

### PR DESCRIPTION
**Pull request by mattnolting**
closes #4407

### CODE REVIEW

File: src/patternfly/components/MenuToggle/examples/MenuToggle.md [hunk 1]
```diff
@@ -143,6 +143,40 @@ import './MenuToggle.css'
 {{/menu-toggle}}
 ```
 
+### Plain with text
+```hbs
+{{#> menu-toggle menu-toggle--IsDisabled="true"}}
```
Comment (mcoker) [inline src/patternfly/components/MenuToggle/examples/MenuToggle.md:148]:
```suggestion
{{#> menu-toggle menu-toggle--IsDisabled="true" menu-toggle--IsPlain="true" menu-toggle--IsText="true"}}
```

File: src/patternfly/components/MenuToggle/examples/MenuToggle.md [hunk 2]
```diff
@@ -143,6 +143,40 @@ import './MenuToggle.css'
...
+    Disabled
+  {{/menu-toggle-text}}
+  {{#> menu-toggle-controls}}
+    {{> menu-toggle-toggle-icon}}
+  {{/menu-toggle-controls}}
+{{/menu-toggle}}
+
+&nbsp;
+
+{{#> menu-toggle menu-toggle--IsExpanded="true" menu-toggle--IsPlain="true" menu-toggle--IsText="true"}}
```
Comment (mcoker) [inline src/patternfly/components/MenuToggle/examples/MenuToggle.md:159]:
```suggestion
{{#> menu-toggle menu-toggle--IsPlain="true" menu-toggle--IsText="true"}}
```

File: src/patternfly/components/MenuToggle/examples/MenuToggle.md [hunk 3]
```diff
@@ -143,6 +143,40 @@ import './MenuToggle.css'
...
+    Custom text
+  {{/menu-toggle-text}}
+  {{#> menu-toggle-controls}}
+    {{> menu-toggle-toggle-icon}}
+  {{/menu-toggle-controls}}
+{{/menu-toggle}}
+
+&nbsp;
+
+{{#> menu-toggle menu-toggle--IsPlain="true" menu-toggle--IsText="true"}}
```
Comment (mcoker) [inline src/patternfly/components/MenuToggle/examples/MenuToggle.md:170]:
```suggestion
{{#> menu-toggle menu-toggle--IsExpanded="true" menu-toggle--IsPlain="true" menu-toggle--IsText="true"}}
```

File: src/patternfly/components/MenuToggle/menu-toggle.scss [hunk 4]
```diff
@@ -132,7 +132,7 @@
     --pf-c-menu-toggle--m-expanded--BackgroundColor: var(--pf-c-menu-toggle--m-primary--m-expanded--BackgroundColor);
   }
 
-  &.pf-m-plain {
+  &.pf-m-plain:not(.pf-m-text) {
```
Comment (mcoker) [inline src/patternfly/components/MenuToggle/menu-toggle.scss:135]: 🔥

**Review by mcoker — CHANGES_REQUESTED** [review]
just a few example updates

**Review by mcoker — APPROVED** [review]
🤩PTM!

**Comment by mcoker** [comment]
@mattnolting can you add this modifier to the docs?

**Comment by mattnolting** [comment]
> @mattnolting can you add this modifier to the docs?

Oh, shoot. Yep, I'll add it!

**Pull request merged**